playwright-e2e/pages/signInPage.page.ts

```diff
@@ -1,20 +1,26 @@
 import { type Locator, type Page } from '@playwright/test';
 
 export default class SignIn {
+  readonly page: Page;
   readonly heading: Locator;
   readonly heading2: Locator;
   readonly email: Locator;
   readonly password: Locator;
   readonly signinButton: Locator;
 
   constructor(page: Page) {
+    this.page = page;
     this.heading = page.getByRole('heading', { name: 'Sign in or create an account' });
     this.heading2 = page.getByRole('heading', { name: 'Sign in', exact: true });
     this.email = page.getByLabel('Email address');
     this.password = page.getByText('Password', { exact: true });
     this.signinButton = page.getByRole('button', { name: 'Sign in' });
-    page.goto(
-      'https://idam-web-public.aat.platform.hmcts.net/login?client_id=adoption-web&response_type=code&redirect_uri=https://adoption-web.aat.platform.hmcts.net/receiver'
+  }
+
+  async navigateTo(): Promise<void> {
+    await this.page.goto(
+      'https://idam-web-public.aat.platform.hmcts.net/login?client_id=adoption-web&response_type=code&redirect_uri=https://adoption-web.aat.platform.hmcts.net/receiver',
```

Comment:
can make URL configurable ?

Reply:
Currently the URL is not configurable, though it would be a good implementation as the e2e test coverage extends throughout adoptions